include/libdnf/base/base.hpp

```diff
@@ -33,10 +37,18 @@ class Base {
 public:
     ConfigMain & get_config() { return config; }
     LogRouter & get_logger() { return log_router; }
+    rpm::RepoSack & get_rpm_repo_sack() { return rpm_repo_sack; }
+    rpm::SolvSack & get_rpm_solv_sack() { return rpm_solv_sack; }
```

Comment:
We have to discuss how other content will be handled like comps, modules in case that they will be not supported.

Reply:
My idea is to add optional code there too. But surrounded by `#ifdef`. Switch on/off support of comps, modules, ... in compile time. Or are you thinking about run time?
Ok, we will discuss it.